lib/fetchers/url.rb

```diff
@@ -153,7 +153,9 @@ def download_archive_to_temp
     def download_archive(path)
       download_archive_to_temp
       final_path = "#{path}#{@archive_type}"
+      FileUtils.mkdir_p(File.dirname(final_path))
       FileUtils.mv(temp_archive_path, final_path)
+      FileUtils.chmod(0644, final_path)
```

Comment:
it shouldn't, but out of abundance of caution, I'll move it out of the fetcher and into the habitat bundle instead.